### ISSUE TITLE
Save state on workspace close, ability to have multiple browsers

### ISSUE THREAD

**Issue by JestDotty** [sean-codes/atom-browser#45]
The use case for this project is probably hot reloading, I am looking for a similar use case though: the ability to do research per project. That way I don't have 120 tabs in my browser. I could just have tabs saved to the project I'm working on.

I'm using a project plugin in conjunction: https://atom.io/packages/project-manager
but this just piggybacks on top of how atom saves project/workspace information (works with other plugins for me) so shouldn't matter

With `atom-browser` if I open the browser dock, close my only project/workspace, and re-open it, the dock is gone. I have CLI/terminal docks from this package (https://atom.io/packages/iv-terminal) and they persist fine though (and I can also make multiples of them with their hotkey to open them)

example of the CLI docks:
![image](https://user-images.githubusercontent.com/4729968/96510572-b6851200-122b-11eb-8773-639e08f56bed.png)

tldr: would you consider my use case and allow spawning multiple browser tabs & saving browser tabs to workspace state?
